client/sites/settings/index.tsx

```diff
@@ -0,0 +1,33 @@
+import page from '@automattic/calypso-router';
+import { makeLayout, render as clientRender } from 'calypso/controller';
+import { siteSelection, sites, navigation } from 'calypso/my-sites/controller';
+import {
+	SITE_SETTINGS,
+	SITE_ADMINISTRATION,
+} from 'calypso/sites/components/site-preview-pane/constants';
+import { siteDashboard } from 'calypso/sites/controller';
+import { siteSettings, administrationSettings } from './controller';
+
+export default function () {
+	page( '/sites/settings', siteSelection, sites, makeLayout, clientRender );
+	page(
+		'/sites/settings/:site',
+		siteSelection,
+		navigation,
+		siteSettings,
+		siteDashboard( SITE_SETTINGS ),
+		makeLayout,
+		clientRender
+	);
+
+	page( '/sites/administration', siteSelection, sites, makeLayout, clientRender );
```

Comment:
Right now, this won't work as we declare the new paths to have `/site/settings` prefix in `/client/sections.js`.

As we'll also have Agency, Caches, and Web Server inside this tab, I'm proposing the following scheme:

- `/sites/settings/site/:site`
- `/sites/settings/administration/:site`
- `/sites/settings/agency/:site`
- `/sites/settings/caches/:site`
- `/sites/settings/web-server/:site`

Other tabs will also follow similar pattern (`/sites/marketing/*/:site`, `/sites/tools/*/:site`)

What do you think?

Reply:
Works for me